frontend/packages/core/src/NPS/feedback.tsx

```diff
@@ -85,7 +85,7 @@ const FeedbackAlert = () => {
     </Alert>
   );
 };
-
+export const FEEDBACK_MAX_LENGTH = 360;
```

Comment:
Just curious, what made you decide on 360? We could also make this tweet size at 280

Reply:
I just thought doubling it would be decent. The individual who gave feedback's response was 264 characters long (it was split across 2 responses) so 280 could work as well.